src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -87,6 +87,11 @@ object Import {
       "bundle-configuration-name",
       "The name of the directory of the additional configuration to use. Defaults to 'default'"
     )
+
+    val compatibilityVersion = SettingKey[String](
+      "bundle-compatibility-version",
+      "A versioning scheme that will be included in a bundle's name that describes the level of compatibility with bundles that go before it. By default we take the major version component of a version as defined by http://semver.org/. However you can make this mean anything that you need it to mean in relation to bundles produced prior to it. We take the notion of a compatibility version from http://ometer.com/parallel.html."
+    )
```

Comment:
@huntc - do we plan for ConductR 1.1 to enforce the presence of this field?

I'm just thinking if ConductR 1.1 enforces the presence of this field, it would mean bundles built with older version of sbt-bundle will be rejected.